src/renderer/wallet/main/index.jsx

```diff
@@ -225,6 +226,10 @@ class Main extends Component {
 						path={`${match.path}/create-corporate-profile`}
 						component={CorporateWizardContainer}
 					/>
+					<Route
```

Comment:
This is will probably needs refactoring later, to move everything corporate related to the /corporate path

Reply:
Sure thing.